syntax = "proto2";

import "ines/third_part/ssl_vision_detection.proto";
import "ines/third_part/ssl_vision_geometry.proto";

message SSL_WrapperPacket {
  optional SSL_DetectionFrame detection = 1;
  optional SSL_GeometryData geometry = 2;
}
